packages/subgraph/.drone.yml

```diff
@@ -48,34 +51,43 @@ steps:
       IPFS_URL:
         from_secret: ipfs-url
     commands:
-      - export SUBGRAPH_NAME=bellecour/dev-dataprotector
+      - export SUBGRAPH_NAME=bellecour/dev-dataprotector-v2
```

Comment:
used `bellecour/dev-dataprotector-v2` instead of `bellecour/dev-dataprotector/v2` (only one `/` is allowed)